mne/fixes.py

```diff
@@ -907,6 +907,64 @@ def get_spectrogram():
 
 
 ###############################################################################
+# Scipy trim_mean (for mne.time_frequency.psd_welch) needed for scipy < 0.12
```

Comment:
If it's only used in PSD you can put it there for now. If you already expect it to be needed elsewhere `utils.py` is fine.